.travis.yml

```diff
@@ -0,0 +1,30 @@
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode9.3
+      language: node_js
+      node_js: '8'
+
+    - os: linux
+      language: node_js
+      node_js: '8'
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - libx11-dev
+            - zlib1g-dev
+            - libpng12-dev
+            - libxtst-dev
+            - g++-4.8
```

Comment:
What Ubuntu version is this, Trusty? Can you try to use the Xenial image? It will have more modern versions of these. Not sure how this affects anything though.

Reply:
Thanks!